str0m/src/media/sender.rs

```diff
@@ -148,15 +149,46 @@ impl SenderSource {
         let rtt = calculate_rtt_ms(ntp_time, r.last_sr_delay, r.last_sr_time);
         self.rtt = rtt;
 
-        self.loss = Some(r.fraction_lost as f32 / u8::MAX as f32);
+        let ext_seq = {
+            let ext = (r.max_seq >> 16) as u64;
+            let num = (r.max_seq & 0xffff) as u16;
+            extend_seq(Some(ext), num)
```

Comment:
The `Some(ext)` part here must be saved on ReceiverSource and reused for every incoming RR.